Lib/zipfile/__init__.py

```diff
@@ -1352,7 +1352,8 @@ class ZipFile:
     mode: The mode can be either read 'r', write 'w', exclusive create 'x',
           or append 'a'.
     compression: ZIP_STORED (no compression), ZIP_DEFLATED (requires zlib),
-                 ZIP_BZIP2 (requires bz2) or ZIP_LZMA (requires lzma).
+                 ZIP_BZIP2 (requires bz2), ZIP_LZMA (requires lzma), or
+                 ZIP_ZSTANDARD (requires zstd).
```

Comment:
```suggestion
                 ZIP_ZSTANDARD (requires compression.zstd).
```

Reply:
PEP 784 mentions:
> New import names `compression.lzma`, `compression.bz2`, `compression.gzip` and `compression.zlib` will be introduced in Python 3.14 re-exporting the contents of the existing `lzma`, `bz2`, `gzip` and `zlib` modules respectively.
> The compression sub-modules will become the canonical import names going forward.

Should I also change the others the reflect this?

Reply:
I think that should wait for when the code in those modules moves. I'll defer to @gpshead on that though.

Reply:
Agreed